src/components/GrampsjsMap.js

```diff
@@ -17,6 +17,8 @@ const defaultConfig = {
   leafletTileSize: 256,
   leafletZoomOffset: 0,
   glStyle: 'https://www.openhistoricalmap.org/map-styles/main/main.json',
+  glAttribution:
+    '<a href="https://www.openhistoricalmap.org/copyright">OpenHistoricalMap</a>',
```

Comment:
OHM data is in the public domain (except where otherwise noted), so this attribution isn’t required, but the project appreciates it as a courtesy. In this case, I figure it would help to keep the Gramps forum from getting confused questions about why this or that isn’t available in the OHM layer. 🙂

Reply:
Yes, makes sense!